docs/modules/pipelines/pages/submitting-jobs.adoc

```diff
@@ -121,6 +121,46 @@ You cannot upload the following classes using the Jet API or the CLI. These clas
 * Map features such as EntryProcessor or MapLoader and MapStore
 ====
 
+=== Job Placement Control
+
+To activate job placement, your license key must include `Advanced Compute`.
+
+You can define which members an individual Jet job runs on. This is known as job placement.
```

Comment:
Maybe one instance where we could mention compute isolation? e.g. 'known as job placement or _compute isolation_' Could understand if we want to omit though it seemed a natural place if we did want to include the term.